src/compiler/Fork.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright 2017-present, Callstack.
+ * All rights reserved.
+ * 
+ * @flow
+ */
+
+import type { Platform } from '../types';
+
+const fs = require('fs');
+const EventEmitter = require('events');
+const Events = require('./events');
+const createForkProcess = require('./createForkProcess');
+const createWebSocketServer = require('./createWebSocketServer');
+
+type ForkConstructorArgs = {
+  platform: Platform,
+  options: *,
+};
+
+const forks = {};
+let transportServer;
```

Comment:
Is there a reason these two are not properties in the `Compiler` instance? Seems like when you create multiple compiler instances from a single process, it's going to reuse the same transport server (with the same address) and overwrite previous forks?

Will probably also be an issue if you had created and terminated a compiler instance previously since this variable is still present, so if you create a new compiler instance, it'll try to use it, but the connection has closed.